src/web/components/SlotBodyEnd.tsx

```diff
@@ -96,14 +118,23 @@ export const SlotBodyEnd = ({
         window.guardian.modules.sentry.reportError(error, 'slot-body-end');
     }
 
-    if (bodyResponse && bodyResponse.data) {
-        // Add a new entry to the view log when we know an Epic is being rendered
-        logView(epicTestName);
-        sendOphanInsertEvent();
+    const [setNode, isVisible] = useVisibility(0.5);
+
+    useEffect(() => {
+        if (isVisible && bodyResponse && bodyResponse.data) {
+            // Add a new entry to the view log when we know an Epic is being rendered
+            throttledLogView(epicTestName);
+            sendOphanEvent('VIEW');
+        }
+    }, [isVisible, bodyResponse]);
```

Comment:
I'd be curious what happens if you don't use a useEffect here. `isVisible` is a state property, as is `bodyResponse` so perhaps you can just put that if statement by itself. It's late on Friday though so I could be missing something obvious

Reply:
Hmm that's a good question. I think you're right. Previously when I only wanted to call `throttledLogView` once I think I needed the `useEffect`. Now I can call many times and that will still only result in one request to Ophan, so maybe I can remove. I'll give it a try!